src/main/java/com/connexta/search/query/QueryManagerImpl.java

```diff
@@ -55,10 +55,9 @@ public QueryManagerImpl(
         uri = new URI(endpointUrlRetrieve + id);
       } catch (URISyntaxException e) {
         throw new QueryException(
```

Comment:
✏️ Should we create a subclass of `QueryException` for when we have communication issues with Solr?

Reply:
Maybe. Let's see how it goes. We'll know better after spending some time with the services in the cloud.